google-cloud-spanner/src/test/java/com/google/cloud/spanner/IntegrationTestEnv.java

```diff
@@ -79,7 +80,7 @@ protected void before() throws Throwable {
     SpannerOptions options = config.spannerOptions();
     String instanceProperty = System.getProperty(TEST_INSTANCE_PROPERTY, "");
     InstanceId instanceId;
-    if (!instanceProperty.isEmpty()) {
+    if (!instanceProperty.isEmpty() && !isUsingEmulator()) {
```

Comment:
This needed to be updated so that the tests run properly with the emulator. A new test instance is created in the emulator.